src/test/java/lv/ctco/javaschool/vote/boundary/VoteApiTest.java

```diff
@@ -57,149 +47,97 @@ void init() {
     }
 
     @Test
-    @DisplayName("Check data for user1,Mood happy = 1,without comments, Date is today")
-    void submitVote_Happy_without_Comment() {
+    @DisplayName("submitDailyVoteTest: Check data for user1,MoodStatus.EMPTY,without comments, Date is today")
+    void submitDailyVote_EMPTY_without_Comment() {
         when(userStore.getCurrentUser())
                 .thenReturn(user1);
-        FeedbackDto feedbackDto = new FeedbackDto();
-        feedbackDto.setMood(1);
-        feedbackDto.setComment("");
+        DailyVoteDto dailyVoteDto = new DailyVoteDto();
+        dailyVoteDto.setMood(MoodStatus.EMPTY);
+        dailyVoteDto.setComment("");
         doAnswer(invocation -> {
             DailyVote dailyVote = invocation.getArgument(0);
-            assertEquals(user1, dailyVote.getUser());
-            assertEquals(LocalDate.now().toString(), dailyVote.getDate());
-            assertEquals(1, feedbackDto.getMood());
-            assertEquals("", feedbackDto.getComment());
+            assertThat(dailyVote.getUser(),equalTo(user1));
+            assertThat(dailyVote.getDate(),equalTo(LocalDate.now()));
```

Comment:
I wonder if this test might fail at some point. The actual date may change between an earlier `dailyVote.setDate()` and this call of `LocalDate.now()`.

Reply:
I guess, if the test is executed near midnight then this test might fail. Watched possible solutions to fix the test, that there is no possibilities to fail. One of the solutions included PowerMockito, maybe there are other suggestions how I could fix this test without using PowerMockito?

Reply:
A solution could be to store the current date in a separate variable, and use that variable both for setting the DTO field and for verification.

Reply:
As discussed - in this case `LocalDate.now()` is called inside the method being tested, so no point in a separate variable. In a real application this could be solved by using a separate service for obtaining the current time, and mocking this service during tests. I'm accepting the PR, since the issue is test-related and is unlikely to appear, so that work can go ahead.